examples/sqlite/wasm/README.md

```diff
@@ -0,0 +1,18 @@
+# `rs-diesel-sqlite`
+
+Diesel's `Getting Started` guide using SQLite instead of Postgresql
+
+## Usage
+
+```
+rustup target add wasm32-unknown-unknown
+# Add wasm32-unknown-unknown toolchain
+
+cargo install wasm-pack
+
+wasm-pack build --web
+# Build wasm
+
+python3 -m http.server 8000
+# Next, use it on the web page
```

Comment:
That should maybe just put a clickable link there, e.g. `[on the web page](http://localhost:8000)` or something like that?